source/cipher_openssl.c

```diff
@@ -811,9 +809,8 @@ int aws_cryptosdk_sig_update(struct aws_cryptosdk_sig_ctx *ctx, const struct aws
 }
 
 int aws_cryptosdk_sig_verify_finish(struct aws_cryptosdk_sig_ctx *ctx, const struct aws_string *signature) {
+    AWS_PRECONDITION(ctx != NULL);
```

Comment:
Why not assert `aws_cryptosdk_sig_ctx_is_valid()`?

Reply:
Updated to this!